krkn/scenario_plugins/pod_disruption/pod_disruption_scenario_plugin.py

```diff
@@ -115,6 +115,22 @@ def get_pods(self, name_pattern, label_selector,namespace, kubecli: KrknKubernet
             pods = kubecli.select_pods_by_name_pattern_and_namespace_pattern(pod_name_pattern=name_pattern, namespace_pattern=namespace, field_selector=field_selector)
         else: 
             logging.error('Name pattern or label pattern must be specified ')
+        
+        # Filter pods by node label selector if specified
+        if node_label_selector and pods:
+            filtered_pods = []
+            nodes_with_label = kubecli.list_nodes(label_selector=node_label_selector)
```

Comment:
let's maybe add an option to list nodes to add in field selector of nodeName and do multiple calls if there are multiple node names (couldn't find a way to add multiple node names in one call)